rpcs3/Emu/Cell/PPUAnalyser.cpp

```diff
@@ -665,8 +665,8 @@ bool ppu_module::analyse(u32 lib_toc, u32 entry, const u32 sec_end, const std::b
 				break;
 			}
 
-			const u32 addr = ptr[0];
-			const u32 _toc = ptr[1];
+			const u32& addr = ptr[0];
```

Comment:
Why is this change?

Reply:
Idk, from experience const ref is faster even for simple types.
Or can it change here?

Reply:
References should be avoided as it's fragile. It may not always result in reference binding. For instance it's not even obvious what it does here.

Reply:
removed